### ISSUE TITLE
`no-unsafe-any` is not flagged

### ISSUE THREAD

**Issue by chriskuech** [Microsoft/vscode-tslint#419]
- tslint Version: 5.12.1
- typescript Version: 3.2.2

Steps to Reproduce:

1. Enable `no-unsafe-any` and `no-any` in your TSLint rules
2. Put the following code in a .ts file
```ts
const o = JSON.parse('{"hello": "world"}');
const s: string = o.myProp;
```
3. Run `tslint --config ./tslint.json --project .`
4. Properly configure VS Code to show linting errors

Observed:

`tslint` command will report `Unsafe use of expression of type 'any'.` while VS Code will not show linting errors on the lines.

**Comment by egamma** [comment]
duplicate, pls see 
https://github.com/Microsoft/vscode-tslint/issues/408#issuecomment-441995969
